system-int-test/src/cmr/system_int_test/utils/association_util.clj

```diff
@@ -98,3 +98,12 @@
         :body
         first
         :tool-association)))
+
+(defn add-individual-statuses
```

Comment:
explain why your creating atoms

Reply:
I was hoping maybe someone knew a better way to do this. But essentially I am taking a list of maps and need to add an item into each map and return the changed list of maps. I did not know how to do this without using atoms in clojure

Reply:
We talked off line, I would suggest that you take the recommended code with a grain of salt and proceed as you see best. I prefer that you do not introduce an atom, but if the code mentioned does not work or causes a delay then roll it back.

Reply:
updated atom funcs to remove atom -- AI for the win this time